test/Game-test.js

```diff
@@ -0,0 +1,78 @@
+const chai = require('chai');
+const expect = chai.expect;
+const Game = require('../js/Game.js');
+const spies = require('chai-spies');
+chai.use(spies);
+global.data = require('../js/data.js');
+global.domUpdates = require('../js/DOM.js');
+global.Round = require('../js/Round.js');
+// global.index = require('../js/index.js');
+chai.spy.on(global.domUpdates, ['getPlayerNames', 'clearInputs', 'goToGameScreen', 'displayWinner', 'goToHomeScreen', 'displayWheel', 'hideWheel'], () => true);
+
+describe('Game', () => {
+  var game;
+  beforeEach(() => {
+    game = new Game()
+  })
+
+  it('should start at round 0', () => {
+  expect(game.round).to.equal(0);
+  expect(game.bonusRound).to.equal(false);
+  });
+
+  it('should start with no players', () => {
+  expect(game.players).to.deep.equal({});
+  });
+
+  it('should start with 4 puzzle banks', () => {
+    expect(game.puzzleKeys).to.deep.equal(['one_word_answers', 'two_word_answers', 'three_word_answers', 'four_word_answers']);
+  });
```

Comment:
You could probably combine these first 3 it blocks into a single block that says `it should have the correct default properties` and have all three/four of these expectations in that single it block.